test_unstructured_ingest/dest/couchbase.sh

```diff
@@ -0,0 +1,105 @@
+#!/usr/bin/env bash
```

Comment:
please make sure this file's permissions are runnable. `chmod 755`

Reply:
Before you check it in, check the permissions locally to see if it is runnable. I believe that works. The next person that downloads the branch should just be able to run it with `./test_unstructured_ingest/dest/couchbase.sh`

If not, I can always do it later.

Reply:
updated